src/libraries/Microsoft.Extensions.Logging.Abstractions/ref/Microsoft.Extensions.Logging.Abstractions.cs

```diff
@@ -202,4 +202,20 @@ public partial class NullLogger<T> : Microsoft.Extensions.Logging.ILogger, Micro
         public bool IsEnabled(Microsoft.Extensions.Logging.LogLevel logLevel) { throw null; }
         public void Log<TState>(Microsoft.Extensions.Logging.LogLevel logLevel, Microsoft.Extensions.Logging.EventId eventId, TState state, System.Exception? exception, System.Func<TState, System.Exception?, string> formatter) { }
     }
+
+    public partial struct BufferedLogRecordMetadata
+    {
+        public DateTimeOffset CreationTime { get; set; }
```

Comment:
in the ref files we always use the fully qualified type names including the namespaces. `System.DateTimeOffset`. This apply to any other additions to this ref file. Thanks!